src/object_store.jl

```diff
@@ -78,3 +80,28 @@ function Base.wait(obj_ref::ObjectRef)
     end
     return nothing
 end
+
+#####
+##### Reference counting
+#####
+
+"""
+    get_all_reference_counts()
+
+For testing/debugging purposes, returns a
+`Dict{ray_jll.ObjectID,Tuple{Int,Int}}` containing the reference counts for each
+object ID that the local raylet knows about.  The first count is the "local
+reference" count, and the second is the count of submitted tasks depending on
+the object.
+"""
+function get_all_reference_counts()
+    worker = ray_jll.GetCoreWorker()
+    counts_raw = ray_jll.GetAllReferenceCounts(worker)
+
+    # we need to convert this to a dict we can actually work with.  we use the
+    # hex representation of the ID so we can avoid messing with the internal
+    # ObjectID representation...
+    counts = Dict(ray_jll.Hex(k) => Tuple(Int.(ray_jll._getindex(counts_raw, k)))
+                  for k in ray_jll._keys(counts_raw))
+    return counts
+end
```

Comment:
Should be moved to `ray_julia_jll`

Reply:
why?  we have lots of functions in Ray.jl that wrap API calls like this and provide the high-level julia interface (i.e., `Ray.put`, `Ray.get`, `Ray.get_job_id`, etc.).  as far as I can tell there aren't really clear criteria for what belongs in Ray.jl and what belongs in the jll.

Reply:
>as far as I can tell there aren't really clear criteria for what belongs in Ray.jl and what belongs in the jll.

we should probably do something about that at some point

Reply:
In this particular case we mainly use this for:

```julia
local_count(oid_hex) = first(get(Ray.get_all_reference_counts(), oid_hex, 0))
```
Which is something I thought we'd want to reuse in the `ray_julia_jll`.

Reply:
> something I thought we'd want to reuse in the ray_julia_jll

maybe?  I don't know off the top of my head what we'd use it for, but if it does become more convenient to define there we can revisit.

if we do need it, it might be more direct to just directly define that via something like

```julia
local_count(oid_hex) = first(_getindex(GetAllReferenceCounts(GetCoreWorker()), FromHex(ObjectID, oid_hex)))
```